commit 1168f1c17336657ad4b29ad682ab40c245c74ea5
Author: madhhur <mm7599355000@gmail.com>
Date:   Fri Mar 7 14:26:48 2025 +0530

    6 files are one to one6

diff --git a/one1.txt b/one1.txt
new file mode 100644
index 0000000..25ab921
--- /dev/null
+++ b/one1.txt
@@ -0,0 +1 @@
+this is one
diff --git a/one2.txt b/one2.txt
new file mode 100644
index 0000000..e69de29
diff --git a/one3.txt b/one3.txt
new file mode 100644
index 0000000..e69de29
diff --git a/one4.txt b/one4.txt
new file mode 100644
index 0000000..e69de29
diff --git a/one5.txt b/one5.txt
new file mode 100644
index 0000000..e69de29
diff --git a/one6.txt b/one6.txt
new file mode 100644
index 0000000..e69de29
